src/http-tequilapi-client.ts

```diff
@@ -143,19 +154,52 @@ export class HttpTequilapiClient implements TequilapiClient {
     return parseConnectionStatisticsDTO(response)
   }
 
-  public async location (timeout?: number): Promise<ConsumerLocationDTO> {
-    const response = await this.http.get('location', undefined, timeout)
+  public async sessionsList (): Promise<SessionDTO[]> {
+    const response = await this.http.get('sessions')
     if (!response) {
       throw new Error('Location response body is missing')
```

Comment:
Out of scope, but there is an issue here
```suggestion
      throw new Error('Session list response body is missing')
```